roxie/ccd/ccdstate.cpp

```diff
@@ -918,18 +918,23 @@ class CRoxieQuerySetManager : public CInterface, implements IRoxieQuerySetManage
     unsigned channelNo;
     bool active;
     StringAttr querySetName;
+    CriticalSection crit;   // For parallel load
 
     void addQuery(const char *id, IQueryFactory *n, hash64_t &hash)
     {
         hash = rtlHash64Data(sizeof(hash), &hash, n->queryHash());
```

Comment:
Were the parameters the wrong way around here?
With this change it should probably say hash = n->queryHash()